scripts/installer/curl_install_pypi/README.md

```diff
@@ -0,0 +1,34 @@
+# Curl Install Script Information
+
+The scripts in this directory are used for installing through curl and they point to the packages on PyPI.
+
+## Install or update promptflow
+
+curl https://promptflowartifact.blob.core.windows.net/linux-install-scripts/install | bash
+
+The script can also be downloaded and run locally. You may have to restart your shell in order for the changes to take effect.
+
+## Uninstall promptflow
+
+Uninstall the promptflow by directly deleting the files from the location chosen at the time of installation. The default install/executable location is the user's home directory ($HOME).
+
+1. Remove the installed CLI files.
+
+   ```bash
+   rm -r <install location>/lib/promptflow
```

Comment:
why not define a variable for install location?

Reply:
Sorry, I'm not sure what do you mean here? I think have added note here like: "The default install/executable location is the user's home directory ($HOME)."

Reply:
updated